canonical-websites/www.ubuntu.com
# Replace autopilot with conjure up on /download/cloud

## Done
Replaced the content on /download/cloud with the new conjure-up version.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/download/cloud](http://0.0.0.0:8001/download/cloud)
- Check that the copy matches the [copy doc](https://docs.google.com/document/d/1y3uaLNn7dxZKcLq2OmVJ96o3NOqn4OTJflty2GmxV9I/edit#)
- Check the page looks ok

## Issue / Card
Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/2182
